packages/modeling/src/operations/extrusions/project.test.js

```diff
@@ -74,9 +75,8 @@ test('project (X and Y axis)', (t) => {
     [1.0000200000000001, -3.999986666666667],
     [0.9238600000000001, -4.382700000000001],
     [0.9807933333333334, -4.1951],
```

Comment:
Any idea why the order of points is changing? I cannot imagine that the order of points in the torus is changing.

Reply:
The torus points do not change order. This just reflects a change in the order of the `sides` of the geom2, the actual shape is the same.

The order of the projected sides change because in `project.js` the geometry is transformed by first a `unionGeom3` and then `unionGeom2`. This converts the geometries to BSP trees, and the output order is affected by very small changes in the input geometry. The shape itself is the same though. (+/- epsilon)